app/controllers/divisions_controller.rb

```diff
@@ -219,4 +233,11 @@ def get_member
           .where(constituency: electorate)
           .order(entered_house: :desc).first
   end
+
+  def get_division(house, date, number)
```

Comment:
Do you think so? I thought this method wasry ve particular to this controller. I extracted as a method to leave the code a bit more readable, but otherwise it would live inside the `show` method. I would rather move if we have repetition. However, I have no strong opinions about it, so if you want me to move, I can move it :smile: